alc/backends/pulseaudio.cpp

```diff
@@ -897,13 +857,12 @@ void PulsePlayback::open(const char *name)
     mStream = mMainloop.connectStream(pulse_name, plock, mContext, flags, nullptr, &spec, nullptr,
         BackendType::Playback);
 
-    pa_stream_set_moved_callback(mStream, &PulsePlayback::streamMovedCallbackC, this);
```

Comment:
This is needed so that a stream that gets moved will stay on the device it moved to after a reset, instead of moving back to the old one.

Reply:
Isn't it expected to reset all moves on reset call?

Reply:
No, `reset` changes/updates the internal playback parameters for the current device with what the app requests. If the playback stream was moved by the server, it should persist through resets. For example, if an app launches and starts playing through speakers by default, the user then uses pavucontrol or kmix or something to move the app playback to headphones, then the app reconfigures/resets the device to enable HRTF, it should keep playing on the headphones, not move back to the speakers.